osf_tests/test_app.py

```diff
@@ -43,6 +43,6 @@ def test_attach_handlers():
     }
 
     # Check that necessary handlers are attached and correctly ordered
-    assert_equal(sorted(set(before_funcs)), sorted(assert_before_funcs))
-    assert_equal(sorted(set(after_funcs)), sorted(assert_after_funcs))
-    assert_equal(sorted(set(teardown_funcs)), sorted(assert_teardown_funcs))
+    assert_equal(set(before_funcs), assert_before_funcs)
+    assert_equal(set(after_funcs), assert_after_funcs)
+    assert_equal(set(teardown_funcs), assert_teardown_funcs)
```

Comment:
I read the comment from https://github.com/CenterForOpenScience/osf.io/blob/develop/website/app.py#L63-L66. I just assume this isn't necessary anymore because we don't use TokuMX. I should remove the comment on ln 45 too.